build.zig

```diff
@@ -172,3 +231,11 @@ pub fn build(b: *Builder) !void {
     });
     debug_step.dependOn(&debug_cmd.step);
 }
+
+fn bash(b: *Builder, comptime fmt: []const u8, args: anytype) *std.build.RunStep {
+    return b.addSystemCommand(&[_][]const u8{
+        "bash",
```

Comment:
As discovered by marler8997, not all systems have bash (#131). I suggest using `b.addSystemCommand` instead.